src/NLog/Config/LoggingRule.cs

```diff
@@ -74,8 +75,8 @@ public LoggingRule(string ruleName)
         /// Create a new <see cref="LoggingRule" /> with a <paramref name="minLevel"/> and  <paramref name="maxLevel"/> which writes to <paramref name="target"/>.
         /// </summary>
         /// <param name="loggerNamePattern">Logger name pattern used for <see cref="LoggerNamePattern"/>. It may include one or more '*' or '?' wildcards at any position.</param>
-        /// <param name="minLevel">Minimum log level needed to trigger this rule.</param>
-        /// <param name="maxLevel">Maximum log level needed to trigger this rule.</param>
+        /// <param name="minLevel">Minimum log level needed to trigger this rule. Log levels range from <see cref="NLog.LogLevel.Trace"/> (lowest) to <see cref="NLog.LogLevel.Fatal"/> (highest).</param>
```

Comment:
Not sure I like to define details LogLevel all around the code. If people need to know of the available LogLevels, then then it should be explained for the LogLevel-type.

Reply:
I added this part because in my opinion there were no sufficient information about what a log level can be, and especially which log levels are high and which are low. In .NET, it's just the other way round: `Trace` is a high log level there, because it produces most log messages. I do not understand why you named your log levels in the opposite way, but of course I need to accept it. But I would add the extra information to make it clear. And I don't see any disadvantages in enhancing the documentation. The XML doc in NLog is quite short, and I think that my additions will help newcomers to understand NLog better and faster (which is the reason why I made all these proposed changes).

Reply:
Yes it is "funny" with LogLevel in .NET Framework. Where [System.Diagnostics.Debug](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.debug) is low (removed on release-builds) and [System.Diagnostics.Trace](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.trace) is high (stays on release-build).

But NLog has inherited its [LogLevels from Log4j](https://www.tutorialspoint.com/log4j/log4j_logging_levels.htm), and now with Microsoft Extension Logging [LogLevel.Trace](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel) it is lowest level again.